packages/core/src/state-management/repository.ts

```diff
@@ -330,6 +338,36 @@ export class Repository {
     return this.#deps.pinStore.stateStore.load(StreamID.fromString(streamID))
   }
 
+  /**
+   * Helper function to add stream to db index if it has a 'model' in its metadata.
+   * @param state
+   * @public
+   */
+  public async indexStreamIfNeeded(state: RunningState): Promise<void> {
```

Comment:
convention is to use `state$` for `RunningState`, and just `state` for `StreamState`

Reply:
ping

Reply:
corrected.